Kentor.AuthServices.TestHelpers/SignedXmlHelper.cs

```diff
@@ -21,7 +21,7 @@ public class SignedXmlHelper
 
         public static string SignXml(string xml)
         {
-            var xmlDoc = new XmlDocument { PreserveWhitespace = true };
+            var xmlDoc = new XmlDocument { PreserveWhitespace = false };
```

Comment:
Even after fixing the main issue with TokenResolver, this change was needed to suppress a "Digest Validation Failed" error in `Saml2SecurityTokenHandler.ReadToken`. The whitespace from the test cases was confusing WIF somehow.